packages/lib-classifier/src/store/UserProjectPreferencesStore.spec.js

```diff
@@ -40,39 +39,49 @@ const authClientStubWithUser = {
   checkBearerToken: sinon.stub().callsFake(() => Promise.resolve(token))
 }
 
-describe('Model > UserProjectPreferencesStore', function () {
+function setupStores (clientStub, authClientStub) {
+  rootStore = RootStore.create({
```

Comment:
Should this function return a new root store? Functions that modify global variables can be confusing.